streams/src/main/java/org/apache/kafka/streams/kstream/internals/foreignkeyjoin/SubscriptionResolverJoinProcessorSupplier.java

```diff
@@ -83,15 +86,10 @@ public void process(final K key, final SubscriptionResponseWrapper<VO> value) {
                 }
                 final ValueAndTimestamp<V> currentValueWithTimestamp = valueGetter.get(key);
 
-                //We are unable to access the actual source topic name for the valueSerializer at runtime, without
-                //tightly coupling to KTableRepartitionProcessorSupplier.
-                //While we can use the source topic from where the events came from, we shouldn't serialize against it
-                //as it causes problems with the confluent schema registry, which requires each topic have only a single
-                //registered schema.
-                final String dummySerializationTopic = context().topic() + "-join-resolver";
```

Comment:
I just missed this last time I made a pass and cleaned up all the pseudo-topics. We should have passed in the common one for hashing LHS values, which is declared in KTableImpl (which is what we're doing now).